base/applications/regedit/edit.c

```diff
@@ -1670,8 +1670,6 @@ BOOL ModifyValue(HWND hwnd, HKEY hKey, LPCWSTR valueName, BOOL EditBin)
         lRet = ERROR_SUCCESS; /* Allow editing of (Default) values which don't exist */
         type = REG_SZ;
         valueDataLen = 0;
-        stringValueData = NULL;
```

Comment:
Why remove this?

Reply:
It is useless/redundant with code that follows.